src/handlers/mentions.rs

```diff
@@ -94,23 +115,36 @@ pub(super) async fn handle_input(
         IssueData::load(&mut client, &event.issue, MENTIONS_KEY).await?;
     // Build the message to post to the issue.
     let mut result = String::new();
-    for to_mention in &input.paths {
-        if state.data.paths.iter().any(|p| p == to_mention) {
+    for (entry, relevant_file_paths) in input.to_mention {
+        if state.data.entries.iter().any(|e| e == &entry) {
             // Avoid duplicate mentions.
             continue;
         }
-        let MentionsPathConfig { message, cc } = &config.paths[to_mention];
+        let MentionsEntryConfig { message, cc, type_ } = &config.entries[&entry];
         if !result.is_empty() {
             result.push_str("\n\n");
         }
         match message {
             Some(m) => result.push_str(m),
-            None => write!(result, "Some changes occurred in {to_mention}").unwrap(),
+            None => match type_ {
+                MentionsEntryType::Filename => {
+                    write!(result, "Some changes occurred in {entry}").unwrap()
```

Comment:
Hmm, we don't actually use the relevant file paths for the filename. But I guess it doesn't hurt to collect them.